packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableCollapsibleDemo.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  SortByDirection,
+  headerCol,
+  TableVariant,
+  expandable,
+  cellWidth,
+  IRow,
+} from '@patternfly/react-table';
+
+export class TableCollapsibleDemo extends React.Component<{}, { columns: any, rows: IRow[] }> {
```

Comment:
Should import `TableProps` and set that as first argument in `React.Component<>`

Reply:
Oops no, nevermind :) Not needed here

Reply:
@jschuler I just pushed a new commit that sets `TableProps` as the first argument there -- did you mean for just this file or for all demos? I can change them all back to use the same syntax as before

Reply:
It looks like the change you suggested also works :)